src/pages/hegel/reference/mechanical-object-section-1.mdx

```diff
@@ -0,0 +1,18 @@
+# The Mechanical Object – Section 1
+
+
+The first section opens with a paragraph that is a reflection on the “mechanical object” rather than a continuation of the immanent development of what the “mechanical object” is, as was the case in the previous paragraph. This is something that Hegel does often, and without any warning, and it is important to distinguish passages that add to the immanent development of a concept from passages that are reflections on that immanent development.  
```

Comment:
Maybe start with a quote? Given you're probably going to do heavy-duty analysis, it might be helpful for the reader to have something to look at.

"case in the previous paragraph. " What previous paragraph? Your next sentence would make much more sense with a bit more context.

Reply:
Would you suggest quoting the full paragraph at the start? I'd be happy to do that.

In this case, the "previous paragraph" refers to the first paragraph of Mechanism that is treated in the first article of the Mechanism series.

Reply:
Yeah, either put up a quote or do a link to it. To make a link, you follow this pattern: 

```
[displayed name](/path/from/root/to-your-article.mdx)

// e.g.
[Hegel's Being](hegel/reference/being)
```

Reply:
What would the link be connected to?

Reply:
What do you mean? You make a link to the previous section? Or do you mean on this article? You could use the word paragraph, since that's what you're referring. Any word can be made a link, whatever's most intuitive. `[paragraph](/my/folder/file)`

Reply:
So just to clarify, by following the above, I would be linking it to a previous article?

Reply:
A link only links exactly where you tell it to. 

Here is a normal, full length link (full length because it includes the https at start). This is what you usually do when you share links externally.

https://www.youtube.com/watch?v=mFjU4JuJgnM

Here is the same link, but formatted with a different display text.

[Click me, baby!](https://www.youtube.com/watch?v=mFjU4JuJgnM)

When you are building a website that has a router, like ours do, we can use _relative links_ . We can drop the https and the domain name and focus only on the path starting with "/", which is known as the root and is usually the home page. Any subsequent words and slashes after "/" define a path, such as "/hegel" or multiple like "/hegel/reference" etc. Have a look at the demo website to see this in action.

Reply:
I've added the block quote at the start of the page. 
 
Regarding the link, would it be like this: src/pages/hegel/reference/mechanical-object.mdx?

Reply:
No, the `src/pages` is not part of the URL.

The URL "begins" inside of the `pages` folder. 

Also, you drop extensions `.mdx` for URLs.